.changes/v3.8.0/921-improvements.md

```diff
@@ -0,0 +1,4 @@
+* Data source `vcd_nsxt_edge_cluster` deprecated `vdc` field in favor of 3 new fields to define
```

Comment:
This is an ask for the PR description:

>  This is a prerequisite for DHCP management in NSX-T Isolated networks

Do we have an issue we could link in the PR description?

Reply:
> Do we have an issue we could link in the PR description?

It was not exactly the same, but very much related to catching up with NSX-T DHCP (vcd_nsxt_network_dhcp) functionality for Org networks ->  https://github.com/vmware/terraform-provider-vcd/issues/907 and https://github.com/vmware/terraform-provider-vcd/issues/885